zoo-css-challenge/index.html

```diff
@@ -90,6 +104,20 @@ <h4>Helping wild turtles in Seychelles</h4>
 				<h4>Learning about the rainforest</h4>
 				<a>Find out More</a>
 			</div>
+			<div class="card">
+				<time>
+					<p id="date"></p>
+					<script>
+						n =  new Date();
+						y = n.getFullYear();
+						m = n.getMonth() + 1;
+						d = n.getDate();
+						document.getElementById("date").innerHTML = y + "-" + m + "-" + d;
+					</script>
```

Comment:
```suggestion
					2021-06-04
```
This task was okay with using plain text like the other cards. Also, it's not common to have `<script>` in the middle of HTML, and we haven't done JS yet in the course so no need to use JS here 😄

Reply:
😅, I did not know how to do without it 

Thank you very much